libs/embed/tools/semantic-release-prepare.ts

```diff
@@ -8,7 +8,6 @@ const pkg = JSON.parse(
 )
 
 pkg.scripts.prepush = "pnpm test:prod && pnpm build"
```

Comment:
<!-- Chores are simple tasks that must be done before the subject can be “officially” accepted. Usually, these comments reference some common process. Try to leave a link to the process description so that the reader knows how to resolve the chore.  -->
☑ **chore:** Well this is annoying... but please add to `cspell` or ignore above to fix this check‏